src/applications/disability-benefits/686c-674/components/ArrayPageItemSpouseTitle.jsx

```diff
@@ -3,9 +3,9 @@ import React from 'react';
 export function SpouseTitle({ formData }) {
   return (
     <div>
-      <h4 className="vads-u-border-color--link-default vads-u-border-bottom--2px vads-u-margin-top--0 vads-u-padding-bottom--0p5">
+      <h3 className="vads-u-border-color--link-default vads-u-border-bottom--2px vads-u-margin-top--0 vads-u-padding-bottom--0p5">
```

Comment:
Will this need a `vads-u-font-size--h4` class to keep the font size consistent ([ref](https://design.va.gov/utilities/font-size))?

Reply:
YES! Good catch